eng/liveBuilds.targets

```diff
@@ -223,4 +224,18 @@
   <PropertyGroup>
     <BundledRuntimeIdentifierGraphFile>$(RuntimeIdGraphDefinitionFile)</BundledRuntimeIdentifierGraphFile>
   </PropertyGroup>
+
+  <Target Name="RewriteRuntimePackDir"
```

Comment:
I'm not thrilled with this approach -- I think we should move to a mechanism to have a "well-known" runtime pack that can be used and preferred over all others. This might require new SDK functionality, I'm not sure.

Also, I'll have to update crossgen to build for net7.0 in that case. Not a big deal since we have a public preview out, but it's an interesting problem for net8.0, where we will have a net8.0 runtime pack, but no bootstrap SDK that would contain 8.0.

Reply:
@jkoritzinsky would appreciate your feedback on this in particular

Reply:
Honestly, this approach (re-pointing to the live runtime pack for the current RID) looks fine to me.

Personally, I'd prefer that we require consuming projects to manually hook this target, especially since it's only used in once place.

When we start work on .NET 8, this should still be handleable without much issue as our work to update framework references .

Maybe we should refactor this into another `.targets` file like `eng/runtimepacks.targets` that can be manually included in projects that will use it (similar to the `eng/targetingpacks.targets`) file.